config/routes.rb

```diff
@@ -368,6 +368,8 @@
   match '/help/unhappy/:url_title' => 'help#unhappy',
         :as => :help_unhappy,
         :via => :get
+  match '/help/unhappy' => 'help#unhappy',
+        :via => :get
```

Comment:
Not a big issue but this route could be combined with the one above as:
```ruby
match '/help/unhappy(/:url_title)' => 'help#unhappy',
```

Then where we're calling `help_general_url(:template => 'unhappy' ...` we could just call `help_unhappy_url(...`